enterprise/utils.py

```diff
@@ -143,6 +143,34 @@ def get_idp_choices():
     return None
 
 
+def get_provider_login_url(request, provider_id):
+    """
+    Return the given provider's login URL.
+    """
+    try:
+        from third_party_auth import pipeline
+    except ImportError as exception:
+        LOGGER.warning("Could not import Pipeline from third_party_auth")
+        LOGGER.warning(exception)
+        pipeline = None  # pylint: disable=redefined-outer-name
+
+    try:
+        from student.helpers import get_next_url_for_login_page
+    except ImportError as exception:
+        LOGGER.warning("Could not import get_next_url_for_login_page from student.helpers")
+        LOGGER.warning(exception)
+        get_next_url_for_login_page = None  # pylint: disable=redefined-outer-name
+
+    third_auth_login_page = ""
+    if pipeline and get_next_url_for_login_page:
+        third_auth_login_page = pipeline.get_login_url(
+            provider_id,
+            pipeline.AUTH_ENTRY_LOGIN,
+            redirect_url=get_next_url_for_login_page(request)
+        )
+    return third_auth_login_page
```

Comment:
what behavior user will face in case this method will return an empty URL?
nit: can we rename `third_auth_login_page` to something line `third_auth_login_page_url` or provider_login_url for clearity.

Reply:
Users will see an error now. I have changed the logic.